cmd/ctl/pkg/inspect/secret/secret_test.go

```diff
@@ -0,0 +1,486 @@
+/*
```

Comment:
Move the tests for functions in util.go, to `util_test.go`

Reply:
they are even not existing yet :D